distribution/pom.xml

```diff
@@ -220,9 +219,45 @@
                             <skipDocker>${docker.skip}</skipDocker>
                             <imageName>wso2/micro-integrator</imageName>
                             <imageTags>
-                                <imageTag>${project.version}</imageTag>
+                                <imageTag>${project.version}-alpine3.10</imageTag>
                             </imageTags>
-                            <dockerDirectory>${basedir}/src/docker-distribution/static</dockerDirectory>
+                            <dockerDirectory>${basedir}/src/docker-distribution/alpine</dockerDirectory>
+                            <buildArgs>
+                                <MICROESB_VERSION>${project.version}</MICROESB_VERSION>
+                            </buildArgs>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>build-image-centos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skipDocker>${docker.skip}</skipDocker>
+                            <imageName>wso2/micro-integrator</imageName>
+                            <imageTags>
+                                <imageTag>${project.version}-centos7</imageTag>
+                            </imageTags>
+                            <dockerDirectory>${basedir}/src/docker-distribution/centos</dockerDirectory>
+                            <buildArgs>
+                                <MICROESB_VERSION>${project.version}</MICROESB_VERSION>
+                            </buildArgs>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>build-image-ubuntu</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                        <configuration>
+                            <skipDocker>${docker.skip}</skipDocker>
+                            <imageName>wso2/micro-integrator</imageName>
+                            <imageTags>
+                                <imageTag>${project.version}-ubuntu18.04</imageTag>
```

Comment:
Do we usually name the ubuntu tags ?

Reply:
We will keep the alpine one as the default one and will name the ubuntu tag as in the PR.